scout/ecm_prep.py

```diff
@@ -14482,48 +14478,19 @@ def main(opts: argparse.NameSpace):  # noqa: F821
                                     p, new_pkg_name,
                                     meas_dict["name"], new_name])
 
-                # Check for whether a reference case analogue measure
-                # should be added (user option is present, measure is in
-                # ESTAR/IECC/90.1 tier, measure applies to equipment
-                # not envelope components
-                if opts is not None and opts.add_typ_eff is True and \
-                    any([x in meas_dict["name"] for x in [
-                        "ENERGY STAR", "ESTAR", "IECC", "90.1"]]) and (
-                        not ((isinstance(meas_dict["technology"], list)
-                              and all([x in handyvars.demand_tech for
-                                       x in meas_dict["technology"]])) or
-                             meas_dict["technology"] in
-                             handyvars.demand_tech)):
+                # Check for whether a reference case analogue measure should be added, which a
+                # user flags via the `ref_analogue` attribute
+                if "ref_analogue" in meas_dict.keys() and meas_dict["ref_analogue"] is True:
```

Comment:
I believe this needs to be:

```suggestion
                if meas_dict.get("ref_analogue") and opts.add_typ_eff:
```

Reply:
Yeah that works to add the check of the `add_typ_eff` option. I guess we assume that no one would set `ref_analogue` to false?